### PR TITLE
Created full website

### PR DESCRIPTION
**Pull request by TuyishimireEric**
## Math magicians: full website
In this project, I have created a full website for the Math magicians app, consisting of several pages and using the components that I already created.
### Project requirements
In previous projects, I created the core functionality of the app: a calculator component. Now I have developed a full website with several pages.

- I created a website consisting of 3 pages: Home, Calculator, and Quote.
- I added my own styles to improve the look and feel of the app.

this is how the project looks like
image
***Cheers and happy coding! :smile_cat:***

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 # Math Magicians: setup
```
Comment (OLIPLICHE) [inline README.md]: # Optional
- Please Update the title to `Math magicians: full website` because it is the title of your project

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @TuyishimireEric , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your App is working fine👏

## Required Changes ♻️
- Please make sure that the general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).
- 
Capture
- Please you can style your App the way you want it but don't change the wireframe
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TuyishimireEric) [reply README.md]: ## Thanks @OLIPLICHE for reviewing my project!
Really my project's general layout was not matching the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).
I have tried my best to match it as required in the project.

***Cheers and happy reviewing!***

File: README.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 - React Js
 - Vs code
 
+## Project Photo
+![Math Magician](./image/pic.png)
+
+
 ## Page Link 
 
 [live link]()
```
Comment (Bondok6) [inline README.md]: - [OPTIONAL] The picture here is not showing up, please fix it 👌💯
- [OPTIONAL] The live demo link is broken, so kindly fix it or remove it for now 👍

> We should always be sure our repositories are documented professionally 💯

image

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @TuyishimireEric 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job using React Router and Styles ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (TuyishimireEric) [reply README.md]: Thank you @Bondok6 for Approving my project 🤗🤗

I appreciate your motivating feedback and I will make the Optional requirements too to make my Readme more professional

**Thanks**

**Pull request merged**